index.html

```diff
@@ -187,26 +187,27 @@ <h2 class="about-grid-item-title">Skills</h2>
         </div>
       </div>
     </div>
-    <div id="contact-div">
-      <div id="contact-center">
-        <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
-        <div id="contact-form-div">
-          <form id = "survey-form">
-            <input id="fullname" class="form name" type="text" placeholder="Full Name" required>
-            <input id="first-name" class="form name desktop" type="text" placeholder="First Name" required><input id="last-name" class="form name desktop" type="text" placeholder="Last Name" required><br>
-            <input id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
-            <textarea id="comments" class="form" name="comments" placeholder="Hey I’d like to connect to..." required></textarea><br>
-            <div id="submit-button" class="grid-button-container">
-              <input type="submit" id="submit" class="grid-button" value="Get in touch">
-            </div>
-            <div id="email-para">
-              <p>nyldrift@hotmail.com</p>
-            </div>
-          </form>
+  </section>
+  <section id="contact">
+    <div id="contact-center">
+      <div id="invite"><p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+      <div id="contact-form-div">
+        <form id = "survey-form" action="https://formspree.io/f/mzbqjbpr" method="POST">
+          <input name="First Name" id="first-name" class="form name" type="text" placeholder="First Name" maxlength="30" required>
+          <input id="Last Name" class="form name desktop" type="text" placeholder="Last Name" required><br>
+          <input name="Email" id="email" type="email" class="form"  placeholder="Email adress" name="email" required><br>
+          <textarea name="Comments" id="comments" class="form" placeholder="Hey I’d like to connect to..." maxlength="500" required></textarea><br>
+          <div id="submit-button" class="grid-button-container">
+            <input type="submit" id="submit" class="grid-button" value="Get in touch">
+          </div>
```

Comment:
- [ ] Please align the button `get in touch` to the center just like that in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=xnorKFPXAqQHUL4d-0)

Figma design:

<img width="314" alt="Screenshot 2023-03-29 at 8 48 43 AM" src="https://user-images.githubusercontent.com/44798044/228425995-da2feda3-4466-470c-b8fe-4cd956ccdd2a.png">

Your design:

<img width="422" alt="Screenshot 2023-03-29 at 8 48 58 AM" src="https://user-images.githubusercontent.com/44798044/228426012-8d2ec860-d150-4b08-857e-86acc84529ea.png">

Reply:
- Button aligned to the center horizontally